packages/react-reconciler/src/__tests__/ReactSuspenseWithNoopRenderer-test.internal.js

```diff
@@ -1653,10 +1732,74 @@ describe('ReactSuspenseWithNoopRenderer', () => {
     // Timeout and commit the fallback
     expect(() => {
       Scheduler.unstable_flushAll();
-    }).toWarnDev(
-      'The following components suspended during a user-blocking update: A, C',
-      {withoutStack: true},
-    );
+    }).toWarnDev('Component was suspended when root was mounted or updated', {
+      withoutStack: true,
+    });
+  });
+
+  it('normal priority updates suspending do not warn for class components', async () => {
+    let setFoo;
+    class App extends React.Component {
+      state = {foo: false};
+
+      render() {
+        setFoo = () => this.setState({foo: true});
+        return (
+          <Suspense fallback="Loading...">
+            {this.state.foo && <AsyncText text="A" />}
+          </Suspense>
+        );
+      }
+    }
+
+    await ReactNoop.act(async () => {
+      ReactNoop.render(<App />);
+    });
+
+    // also make sure lowpriority is okay
+    await ReactNoop.act(async () => {
+      Scheduler.unstable_runWithPriority(
+        Scheduler.unstable_NormalPriority,
+        () => setFoo(true),
+      );
+    });
+
+    expect(Scheduler).toHaveYielded(['Suspend! [A]']);
+    Scheduler.unstable_advanceTime(100);
+    await advanceTimers(100);
+
+    expect(Scheduler).toHaveYielded(['Promise resolved [A]']);
+  });
+
+  it('normal priority updates suspending do not warn for functional components', async () => {
+    let _setFoo;
+    function App() {
+      let [foo, setFoo] = React.useState(false);
+      _setFoo = setFoo;
+      return (
+        <Suspense fallback="Loading...">
+          {foo && <AsyncText text="A" />}
+        </Suspense>
+      );
+    }
+
+    await ReactNoop.act(async () => {
+      ReactNoop.render(<App />);
+    });
+
+    // also make sure lowpriority is okay
+    await ReactNoop.act(async () => {
+      Scheduler.unstable_runWithPriority(
+        Scheduler.unstable_NormalPriority,
```

Comment:
Normal is the default priority (we're probably going to rename it to DefaultPriority, in fact), so you can remove this part of the test